### PR TITLE
Fix `edit` section in UG

### PR DESCRIPTION
**Pull request by rylzxc**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ What you will see:
 
 |                                                  |
 |:------------------------------------------------:|
-| ![AddCommandResult](images/AddCommandResult.png) ||
+| ![AddCommandResult](images/AddCommandResult.png) |
 
 
 <div markdown="block" class="alert alert-success">
```
Comment (jacobkwan) [inline docs/UserGuide.md:197]: is this what you intended? just checking
Screenshot 2022-11-06 at 11 10 15 PM

Comment (rylzxc) [reply docs/UserGuide.md:197]: yup, otherwise it appears as 2 columns and the one on the right will have no contents

Comment (jacobkwan) [reply docs/UserGuide.md:197]: oh but what about the empty header? based on the other tables I'm guessing you wanted to put "What you will see" inside as the header?

Comment (jacobkwan) [reply docs/UserGuide.md:197]: Screenshot 2022-11-07 at 12 02 47 AM

Comment (rylzxc) [reply docs/UserGuide.md]: resolved

**Review by jacobkwan — APPROVED** [review]
LGTM

**Pull request merged**